tasks/kyu8.py

```diff
@@ -4,14 +4,14 @@
 
 """
 
-
-def example(first, second):
-    """calculate sum a, b.
+def fix_the_meerkat(arr):
+    """
     Args:
```

Comment:
Add a line in function docstring before "Args: " which tells what this function does